### PR TITLE
First set up bookstore

### PR DESCRIPTION
**Pull request by J-C-S-V**
Hello dear reviewer 👋, I hope you're having a nice day today 😊

This is what I did on this branch:

- Setup a new React application using Create React App
- Create a directory for your reusable components: /components.
- Install [React Router V6](https://reactrouter.com/en/main/start/tutorial#setup)
- Setup your router inside of App
- Add 2 routes in your application, each should render a page component: ( /  ) - ( /categories )
- Create components inside of the /components directory

Best regards! ✨

### CODE REVIEW

**Review by alexpuente0 — COMMENTED** [review]
Hi @J-C-S-V , 



While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request 🟠 🟠 🟠 

You have submitted a project with a Readme file that is from other project. Please update the information on your Readme file so it matches the information from this project, and kindly re-submit for a review. Make sure your Readme file looks professional.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

File: src/components/BookCard.jsx [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+          <p className="book__left-p">Suzanne Collins</p>
+          <ul className="book__left-ul">
+            <li className="book__left-li">Comments</li>
+            <li className="book__left-li">Remove</li>
+            <li className="book__left-li">Edit</li>
+          </ul>
+        </div>
+        <div className="book__left-circle">
+          <svg
+            className="book__left-svg"
```
Comment (Shedrack-Sunday) [inline src/components/BookCard.jsx]: - [x] You actually did great work here 👏 . However, i'll advice you breakdown this component into two sections.

- `<Book>`: Takes the book data as a prop and renders the title and the author
  * Has a button for removing books
  * the button wouldn't remove the books *yet*, this is just a placeholder for now

- `<BookList>`: Renders a list of books using the `<Book>` component

- [x] Consider using your `form` component here also, so you have a composite `Bookcard` component to call anywhere in your code. This is the idea of react after all, to have reusable component 👍

**Review by Shedrack-Sunday — CHANGES_REQUESTED** [review]
Hello @J-C-S-V        👋 

# CHANGES REQUIRED ♻️

Great work on the project 👏
You've done well so far in the project,  However, there are a few you need to work on, to move to the next project. 💯

 


## Highlights   
- Good use of React libraries ✔️   
- Nice file struture 👍 
- Beautiful UI ✔️ 
  
# Required Changes  🌵 

- [ ] I made suggestions around your `BookCard` component implementation. Kindly follow the suggestion i made to have a very professional looking work of  which your code looks good so far 👍 

_Check the comments for a detailed review 

# Optional suggestions 

Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: src/components/BookCard.jsx (hunk 1 above)
Comment (J-C-S-V) [reply src/components/BookCard.jsx]: Hello @Shedrack-Sunday,

When you say "into two sections" at the beginning you mean components?

**Review by OyePriscilla — APPROVED** [review]
Hi @J-C-S-V 👏 👏, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉
 


## Highlights
Initialized project with CRA ✔️ 
Use React components ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by J-C-S-V** [comment]
@OyePriscilla Thank you!!!!!

**Pull request merged**